README.md

```diff
@@ -105,8 +103,9 @@ Something to be aware of as a Rubyist learning JavaScript is that JavaScript wil
 # => TypeError: no implicit conversion of Fixnum into String
 ```
 
-Unlike Ruby, JavaScript will convert this final number into a string and tack that string onto the end:
+Unlike Ruby, JavaScript will convert this final number into a string and tack that string onto the end.
 
+Javascript:
 ```javascript
 // JavaScript
 
```

Comment:
yea i think it's confusing here - maybe put it in a later readme if math with money comes up?